src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java

```diff
@@ -218,7 +218,13 @@ public static ItemStack getItemStack(JsonObject json, JsonContext context)
             // Lets hope this works? Needs test
             try
             {
-                NBTTagCompound nbt = JsonToNBT.getTagFromJson(GSON.toJson(json.get("nbt")));
+                JsonElement element = json.get("nbt");
+                NBTTagCompound nbt;
+                if(element.isJsonObject())
```

Comment:
since the code is a bit inconsistent, i asked on IRC on how to handle it, this is mezz's repsonse:
> [20:27:24] <+mezz> Xalcon, you are "allowed" to leave braces off of one-line it/else bodies. personally I choose not to do that most of the time. it's up to you

But I'll change it if mezz or lex wants this to have { } :)